config/crds/bases/external-secrets.io_clusterexternalsecrets.yaml

```diff
@@ -302,6 +302,34 @@ spec:
                             Multiple Rewrite operations can be provided. They are applied in a layered order (first to last)
                           items:
                             properties:
+                              merge:
+                                description: |-
+                                  Used to merge key/values in one single Secret
+                                  The resulting key will contain all values from the specified secrets
+                                properties:
+                                  conflictPolicy:
+                                    default: Ignore
+                                    description: Used to define the policy to use
+                                      in conflict resolution.
+                                    type: string
```

Comment:
```suggestion
                                  conflictPolicy:
                                    default: Error
                                    description:  |-
                                      Defines how to handle key conflicts when merging:
                                      - 'Error': fail if the same key appears more than once.
                                      - 'Ignore': accept the first value found, ignore subsequent ones.
                                      - 'Overwrite': overwrite previous values with later ones.
                                    enum:
                                    - Error
                                    - Ignore
                                    - Overwrite
                                    type: string
```

Reply:
Hey Riccardo, so nice to see a familiar face here! Maybe that is why I feel bold enough to chime in.

The `conflictPolicy` defaulting to "Ignore" is risky, it allows silent key overwrites. I recommend changing the default to "Error", so that we fail on conflicts, and adding an enum to catch typos and enforce safe behaviors. What do you think?

Reply:
`Error` is now the default `conflictPolicy`. I decided against adding `Overwrite` because it makes the logic too complicated to follow and you can obtain pretty much the same behavior by using priority.